dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.43/V2_43_24__migrate_old_tracker_search_optimization_job.sql

```diff
@@ -0,0 +1,3 @@
+update jobconfiguration
```

Comment:
I don't think this will work. 
Since we are not migrating the jobparameters from the old job to the new job. 

For this migration, simply deleting the old job configuration would suffice.